sphinx/writers/latex.py

```diff
@@ -399,6 +401,15 @@ def __init__(self, document, builder):
         if self.elements['polyglossia']:
             self.elements['babel'] = ''  # disable babel forcely
             self.elements['multilingual'] = self.elements['polyglossia']
+            self.elements['classoptions'] += ',' + self.babel.get_language()
+            if builder.config.language:
```

Comment:
Thus, I conclude that if `language` variable is not explicitely set by user, then `\setmainlanguage{english}`  should be used. Does not `self.babel.get_language()` return the string ``'english'`` by default ?